docs/execplans/8-2-3-message-key-support.md

```
@@ -0,0 +1,431 @@
+- `src/message_assembler/series.rs` (~150 lines)
+- `src/message_assembler/state.rs` (~200 lines)
+- `tests/features/message_assembly.feature` (~60 lines)
+- `tests/worlds/message_assembly.rs` (~150 lines)
+- `tests/steps/message_assembly_steps.rs` (~100 lines)
+
+## Interfaces and Dependencies
+
+At completion, the following public interfaces must exist in
+`wireframe::message_assembler`:
+
+    pub enum MessageSeriesStatus {
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)